hack/chains/config.sh

```diff
@@ -67,8 +67,8 @@ case "$1" in
     ;;
 
   rekor-local )
-    REKOR_SERVER=$(kubectl get ingress -n rekor-server -o yaml|yq e '.items[.spec].spec.rules[.host].host' -)
-    $0 "transparency.url: \"https://$REKOR_SERVER\"" $2
+    REKOR_SERVER=$(kubectl get ingress -n rekor -o jsonpath='{.items[0].spec.rules[0].host}')
+    $0 "transparency.url: \"http://$REKOR_SERVER:3000\"" $2
```

Comment:
It seems strange that we need to hard code an internal service port here, so I'm wondering if we
re on the wrong track..

Reply:
I believe the reason is that the deployment of this helm operator uses '3000' for both the `port`  and the `targetPort` where, iiuc, a service will often use '80' as the `port` and then have the `targetPort` to whatever is need, '3000' in this case.